src/PointwiseFunctions/GeneralRelativity/Ricci.cpp

```diff
@@ -50,6 +50,43 @@ tnsr::aa<DataType, SpatialDim, Frame, Index> ricci_tensor(
                d_christoffel_2nd_kind);
   return result;
 }
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+void ricci_scalar(
```

Comment:
I believe this function is identical to `trace` in `IndexManipulation.hpp`. I suggest you use `trace` in this implementation, or even consider removing `ricci_scalar` entirely and just use `trace`? @geoffrey4444 what do you think?